### PR TITLE
CHK-1611-api-key-for-test

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Added header **Ocp-Apim-Subscription-Key** into collection for set api-key and set the correct value when running tests after deploying.

**Pipeline CR with default api key defined into template:** https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=99279&view=logs&j=edf6994b-8050-56ab-f853-4abd46a53714 

Deploy DEV pipeline

#### Motivation and Context
The change was made to set the correct value based on the api-key environment during post deployment tests.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api-tests/ACA-api.tests.json [hunk 1]
```diff
@@ -28,6 +28,11 @@
 						"key": "deployment",
 						"value": "{{DEPLOYMENT}}",
 						"type": "text"
+					},
+					{
+						"key": "Ocp-Apim-Subscription-Key",
+						"value": "{{API_SUBSCRIPTION_KEY}}",
+						"type": "text"
```
Comment (pietro-tota) [inline api-tests/ACA-api.tests.json:35]: ### 
```suggestion
						"type": "secret"
```

File: api-tests/env/ACA_dev_env.json [hunk 2]
```diff
@@ -31,6 +31,12 @@
 			"value": "aca/v1/paCreatePosition",
 			"type": "default",
 			"enabled": true
+		},
+		{
+			"key": "API_SUBSCRIPTION_KEY",
+			"value": "default-key",
+			"type": "default-key",
```
Comment (pietro-tota) [inline api-tests/env/ACA_dev_env.json:38]:
```suggestion
			"type": "secret",
```

File: api-tests/env/ACA_uat_blue_env.json [hunk 3]
```diff
@@ -31,6 +31,12 @@
 			"value": "blue",
 			"type": "blue",
 			"enabled": true
+		},
+		{
+			"key": "API_SUBSCRIPTION_KEY",
+			"value": "default-key",
+			"type": "default-key",
```
Comment (pietro-tota) [inline api-tests/env/ACA_uat_blue_env.json:38]:
```suggestion
			"type": "secret",
```

File: api-tests/env/ACA_uat_env.json [hunk 4]
```diff
@@ -31,6 +31,12 @@
 			"value": "aca/v1/paCreatePosition",
 			"type": "default",
 			"enabled": true
+		},
+		{
+			"key": "API_SUBSCRIPTION_KEY",
+			"value": "default-key",
+			"type": "default-key",
```
Comment (pietro-tota) [inline api-tests/env/ACA_uat_env.json:38]:
```suggestion
			"type": "secret",
```

**Review by pietro-tota — APPROVED** [review]
Well done, just some trivial issues

File: api-tests/env/ACA_dev_env.json (hunk 2 above)
Comment (Fernando-Granato) [reply api-tests/env/ACA_dev_env.json]: Fixed with 50f393c2f3a58b807c19d44801950f6919a6ab8a 🙂

**Review by pietro-tota — APPROVED** [review]

File: api-tests/env/ACA_uat_blue_env.json (hunk 3 above)
Comment (Fernando-Granato) [reply api-tests/env/ACA_uat_blue_env.json]: Fixed with 50f393c2f3a58b807c19d44801950f6919a6ab8a 🙂

File: api-tests/ACA-api.tests.json (hunk 1 above)
Comment (Fernando-Granato) [reply api-tests/ACA-api.tests.json]: Fixed with 6c64f095c36441656675138484c0c82564c6cc07 🙂

**Review by infantesimone — APPROVED** [review]

**Pull request merged**